dashboard/app/controllers/courses_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
     @recent_courses = current_user.try(:recent_courses)
     @is_teacher = !!(current_user && current_user.teacher?)
     @is_english = request.language == 'en'
+    @is_signed_out = current_user.nil?
```

Comment:
I thought about it both ways, and to my brain it seemed clearer to have signed_out because it felt more separate from is_teacher because you can only be a teacher if you're signed in. This way I don't have to check for the opposite of the condition in which I need to do something special.